app/pandas.ipynb

```diff
@@ -549,14 +1389,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
```

Comment:
The Pandas solution for calculating the moving average is missing. Implement it using `data_series.rolling(window=5).mean().dropna()` to meet the task requirements.